mountpoint-s3/CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 ## Unreleased
 
+## v1.3.2 (January 11, 2024)
+
+### Breaking changes
+* No breaking changes.
+
+### Other changes
+
```

Comment:
nit: probably remove this empty line, just want it to be be consistent with the rest of the file.